inst/templates/ad_advs.R

```diff
@@ -107,7 +107,9 @@ advs <- advs %>%
     AVALC = VSSTRESC
   ) %>%
 
-  # Derive new parameters based on existing records.
+  # Derive new parameters based on existing records. Note that, for the following
+  # three derive_param_* functions, only the variables specified in `by_vars` will
+  # be populated in the newly created records.
```

Comment:
Please put `derive_param_*()` in backticks add add a set of trailing parentheses.

Reply:
Yes of course, I will do this now. Is this just in the template programs or shall I check the vignettes also?